mms/tests/integration_tests/test_utils.py

```diff
@@ -46,6 +46,13 @@
 }
 
 
+def filtered_urls(model_list, url):
```

Comment:
dict comprehension would be simple and clearer. the whole method body can be replaced with:

```
return {model: url[model] for model in model_list}
```

also 'url' is a confusing name for a dict of names -> urls. model_data or model_urls are better